models/user.py

```diff
@@ -38,8 +38,7 @@ class UserModel(BaseModel):
     firstName = db.Column(db.String(100), nullable=False)
     lastName = db.Column(db.String(100), nullable=False)
     phone = db.Column(db.String(20), nullable=False)
-    hash_digest = db.Column(db.LargeBinary(60))
-    password = db.Column(db.String(), default=None, onupdate=None)
+    _password = db.Column("password", db.LargeBinary(60))
```

Comment:
What does this do `db.Column("password", db.LargeBinary(60)`?
I'm not familiar with passing a string as the first argument.

Reply:
String as the first argument, which will be the name of the field in the table.